packages/webpack-config/webpack.config.js

```diff
@@ -11,23 +9,16 @@ function invokePossibleFunction(objectOrMethod, ...args) {
   }
 }
 
-module.exports = function(env = {}, argv) {
-  // Check if the project has a webpack.config.js in the root.
-  const projectWebpackConfig = path.resolve(process.cwd(), 'webpack.config.js');
-  if (fs.existsSync(projectWebpackConfig)) {
-    const webpackConfig = require(projectWebpackConfig);
-    return invokePossibleFunction(webpackConfig, env, argv);
+function getConfigFromRootIfPossible(filename) {
```

Comment:
Do we still need this here? Since you can have `webpack.config.js` people can structure their config anyway they want, if they override it. That would keep the `webpack.dev.js` and `webpack.prod.js` filenames as an implementation detail too, so we can also change the structure internally, if needed.

Reply:
this makes sense. I think originally the idea was to make it easier to edit the prod output, but I think the logic is getting somewhat crippling.